addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1355,6 +1355,9 @@ const ColorpickerUserValueWidget = SelectUserValueWidget.extend({
             options.withCombinations = !!this.options.dataAttributes.withCombinations;
         }
         const oldColorPalette = this.colorPalette;
+        if (!oldColorPalette) {
+            options.neverShown = true;
+        }
```

Comment:
Mmh, this commit feels strange, especially without comments. If we indeed don't care about the first `colorPalette` render, then there is probably a better way by not rendering it at all and `refactore` the re-render method a bit to know where to put the re-rendered widget. And load the `colorPalette` dependencies explicitly in the `UserValueWidget`. Anyways, if you manage to do what I explained for the second commit, this third commit would not be necessary at all anymore though :wink: